package/abs/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:16.04
 RUN apt-get update && \
     apt-get install -y jq python2.7 python-pip curl
-RUN pip install --upgrade pip && \
```

Comment:
Changes in pip have broken this behavior. Builds will fail without updating this, locking to a specific version, or removing the update entirely.